baize/datastructures.py

```diff
@@ -239,15 +239,15 @@ def __repr__(self) -> str:
         return f"{self.__class__.__name__}({repr(url)})"
 
 
-class MultiMapping(typing.Generic[KT, VT], typing.Mapping[KT, VT]):
+class MultiMapping(typing.Mapping[KT, VT]):
```

Comment:
If `Generic` is removed, usage like `MultiMapping[str, str]` cannot be used.

Reply:
Why not, built-in `typing.Protocol` is such type -- it is generic, and `typing.SupportsAbs` is still generic and it only derives from `Protocol[T_co]` since `T_co` can be given a concrete type in subclasses.

Reply:
You can try. Just run pytest